Cargo.toml

```diff
@@ -17,31 +18,34 @@ travis-ci = { repository = "pingcap/rust-prometheus" }
 features = ["nightly"]
 
 [features]
-default = ["protobuf"]
 nightly = ["libc"]
 push = ["reqwest", "libc"]
 process = ["libc", "procinfo"]
-gen = ["protobuf-codegen-pure"]
+gen = ["protobuf-build"]
 
 [dependencies]
-protobuf = { version = "2.0", optional = true }
+protobuf = "2.0"
```

Comment:
Can you keep it optional? See https://github.com/pingcap/rust-prometheus/pull/216